core-web/apps/dotcms-binary-field-builder/tsconfig.spec.json

```diff
@@ -3,7 +3,8 @@
     "compilerOptions": {
         "outDir": "../../dist/out-tsc",
         "module": "commonjs",
-        "types": ["jest", "node"]
+        "types": ["jest", "node"],
+        "target": "es2016"
```

Comment:
Should we update to `ECMAScript 2021` in the `tsconfig.base.json`? Since that's the latest stable version

Reply:
changed to 2021 and works fine.